### PR TITLE
App Submission: Jackett

### PR DESCRIPTION
**Pull request by IMPranshu**
This PR takes care of the issue https://github.com/getumbrel/umbrel-apps/issues/487
But additional testing is needed. I think we may need to whitelist something in order to get it completely. Anyways I have no idea need some reviews

# App Submission

### Jackett
...

### 256x256 SVG icon
https://raw.githubusercontent.com/IMPranshu/svg-logo/main/jackett.svg
...

### Gallery images

jackett-screenshot3
jackett-screenshot2
jackett-screenshot1

...

### I have tested my app on:
- [x] [Umbrel dev environment](https://github.com/getumbrel/umbrel-dev)
- [x] [Umbrel OS on a Raspberry Pi 4](https://github.com/getumbrel/umbrel-os)
- [x] [Custom Umbrel install on Linux](https://github.com/getumbrel/umbrel#-installation)

### CODE REVIEW

**Comment by nmfretz** [comment]
>  I think we may need to whitelist something in order to get it completely. Anyways I have no idea need some reviews

Thanks @IMPranshu, I'll look into this

File: jackett/umbrel-app.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+permissions:
+  - STORAGE_DOWNLOADS
+torOnly: false
+submitter: Pranshu Agrawal
+submission: ""
```
Comment (ParthJadhav) [inline jackett/umbrel-app.yml:25]: @IMPranshu , Please add link to this PR in the submission.

**Review by ParthJadhav — CHANGES_REQUESTED** [review]
Hey @IMPranshu , It works on `umbrel-dev`. 

There's only one change i've mentioned above otherwise LGTM !

Thanks for the contribution ❤️

**Review by ParthJadhav — APPROVED** [review]
LGTM !

**Comment by ParthJadhav** [comment]
Hey @IMPranshu, You asked about white listing directories at the start of the PR. Sorry I missed that. 

Regarding that:
If you go to this page: https://hub.docker.com/r/linuxserver/jackett/

and look at the `docker-compose.yml` in the `usage` section you'll find:

```yml
--
version: "2.1"
services:
  jackett:
    image: lscr.io/linuxserver/jackett:latest
    container_name: jackett
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/UTC
      - AUTO_UPDATE=true #optional
      - RUN_OPTS= #optional
    volumes:
      - /path/to/data:/config
      - /path/to/blackhole:/downloads
    ports:
      - 9117:9117
    restart: unless-stopped
```

Here it's suggested that you mount two volumes:
```yml
 volumes:
      - /path/to/data:/config
      - /path/to/blackhole:/downloads
```

If you want to do it with Umbrel then you will need to mount Umbrel's path:
```yml
 volumes:
      - ${APP_DATA_DIR}/data:/config
      - ${APP_DATA_DIR}/blackhole:/downloads
```

After doing that you would also need to push those directories along with the changes. 
To push empty directories you would include `.gitkeep` file inside the directory.

```
- jackett 
-- docker-compose.yml
-- umbrel-app.yml
-- data
--- .gitkeep
-- blackhole
--- .gitkeep
```

**Comment by nmfretz** [comment]
Thanks again for submitting Jackett @IMPranshu. And thanks for your review @ParthJadhav. This is going live to the app store now 🥼.

To bring this to the finish line, we have:
- changed the port to Jackett's default port (9117) so that users who search for troubleshooting tips online don't need to do any translations
- bind mounted the blackhole directory to the Umbrel downloads directory on the host
- added the data directory to this repo to ensure proper ownership/permissions when the app is installed
- bumped Jackett to the latest version
- whitelisted the api route in the app proxy
- added some set up instructions

**Pull request merged**